Gems/Atom/RPI/Code/Include/Atom/RPI.Public/ViewGroup.h

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <Atom/RPI.Public/View.h>
+#include <Atom/RPI.Public/ViewProviderBus.h>
+#include <Atom/RPI.Public/XR/XRRenderingInterface.h>
+#include <AzCore/Name/Name.h>
+
+namespace AZ
+{
+    namespace RPI
+    {
+        //! This class is responsible for managing stereoscopic and non-stereoscopic views.
+        //! It also allows calling code to provide handlers or functions to be called in case the projection
+        //! or view matrix changes. 
+        class ViewGroup final
+        {
+        public:
+            AZ_TYPE_INFO(View, "{C1AD45ED-03E2-41E0-9DC7-779B4B1842FF}");
```

Comment:
Wrong name passed to `AZ_TYPE_INFO`, `View` -> `ViewGroup`.

Please double check the UUID is unique, maybe this was a copy/paste and forgot to update it.